### PR TITLE
PSDK-28 unified api client

### PR DESCRIPTION
**Pull request by hkad98**

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

File: Makefile [hunk 1]
```diff
@@ -38,22 +38,45 @@ format-fix:
...
+	$(call download_generate_client,scan)
+
+.PHONY: api-client
+api-client: download
+	rm -f schemas/gooddata-api-client.json
+	cat schemas/gooddata-*.json | jq -S -s 'reduce .[] as $$item ({}; . * $$item) + { tags : ( reduce .[].tags as $$item (null; . + $$item) | unique_by(.name) ) }' > "schemas/gooddata-api-client.json"
+	$(call generate_client,api)
+
+.PHONY: download
+download:
```
Comment (pcerny) [inline Makefile:76]: Generate also exporter endpoints to the unified api client. See https://staging.anywhere.gooddata.com/apidocs/?urls.primaryName=Export%20(v1).

File: scripts/generate_client.sh [hunk 2]
```diff
@@ -83,6 +83,9 @@ case "$GD_API_CLIENT" in
     GD_API_URI_PATH="${GD_SCHEMA_URL}/api/${API_VERSION}/schemas/afm"
```
Comment (pcerny) [inline scripts/generate_client.sh:83]: Improve usage help. Add there that `gooddata-api-client` as supported option.

File: gooddata-api-client/LICENSE.txt [hunk 3]
```diff
@@ -0,0 +1 @@
+../OSS LICENSES/LICENSE (gooddata-api-client).txt
```
Comment (pcerny) [inline gooddata-api-client/LICENSE.txt:1]: It looks like invalid symlink. I do not see `OSS LICENSES/LICENSE (gooddata-api-client).txt` being part of PR.

**Review by pcerny — CHANGES_REQUESTED** [review]

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Review by pcerny — APPROVED** [review]

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-python-sdk-gate-PR155

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Pull request merged**